bundles/mapping/layerswipe/publisher/SwipeTool.js

```diff
@@ -0,0 +1,76 @@
+import { AbstractPublisherTool } from '../../../framework/publisher2/tools/AbstractPublisherTool';
+import { SwipeToolhandler } from '../handler/SwipeToolHandler';
+import { SwipeToolComponent } from './SwipeToolComponent';
+
+export const SWIPE_ID = 'Oskari.mapframework.bundle.layerswipe.plugin.LayerSwipePlugin';
+
+class SwipeTool extends AbstractPublisherTool {
+    constructor (...args) {
+        super(...args);
+        this.index = 5;
+        this.group = 'additional';
+        this.handler = new SwipeToolhandler(this);
+    }
+
+    getTool () {
+        return {
+            id: SWIPE_ID,
+            title: Oskari.getMsg('LayerSwipe', 'tool.label'),
+            config: this.state.pluginConfig || {}
+        };
+    }
+
+    getComponent () {
+        return {
+            component: SwipeToolComponent,
+            handler: this.handler
+        };
+    }
+
+    init (data) {
+        const configuration = data?.configuration?.layerswipe || {};
+        if (configuration?.state?.active) {
+            this.setEnabled(true);
+        }
```

Comment:
setEnabled(true) is now called here and a few lines after again. Can we remove the other/this one?